addons/tagging/CfgVehicles.hpp

```diff
@@ -4,57 +4,31 @@ class CfgVehicles {
         class ACE_SelfActions {
             class ACE_Equipment {
                 class ACE_tagWallBlack {
```

Comment:
Should we rename the classes or rather keep backwards compatibility? They are no longer "wall" specific.